ovs-dpdk.md

```diff
@@ -0,0 +1,170 @@
+# oVirt now supports OVS-DPDK
+
+
+[DPDK](http://dpdk.org/) (Data Plane Development Kit) provides high-performance packet processing libraries and user space drivers.
+Open vSwitch supports DPDK since v2.2 and starting with version 2.4, a DPDK-optimized vHost path is enabled in OVS.
+ 
+According to [Intel ONP performance tests](https://download.01.org/packet-processing/ONPS2.1/Intel_ONP_Release_2.1_Performance_Test_Report_Rev1.0.pdf), using OVS with DPDK can provide a huge performance enhancement, increasing network packets throughput and reduced latency.
+ 
+OVS-DPDK has been added to oVirt as an experimental feature.
+Please note that OVS-DPDK cannot access any features located within the linux kernel, for example: linux bridge,  tun/tap devices, iptables etc.
+
+## Requirements:
+
+For best performance use an Intel Xeon class server system such as Ivy Bridge, Haswell or newer.
+Ensure that each memory channel has at least one memory DIMM inserted, and that memory size should be at least 4GB.
```

Comment:
That is extremely fine grained recommendation, do you have a source you could add to the post?